### DISCUSSION TITLE
Feedback Widget in Docs

### DISCUSSION THREAD

**Discussion by AleksandarCole** [semaphoreci/semaphore#52] [Documentation improvements]
# Feedback Widget in Docs

| Authors       | State    |
|---------------|----------|
| @AleksandarCole <br> **looking for contributor**  | `Draft`   |

## What
We can implement a feedback widget integrated in the documentation pages similar to this:
![image](https://github.com/semaphoreci/semaphore/assets/61409859/d68ce935-f4c9-4d35-8810-b8ade93ffea4)

Readers could use this widget to rate the pages and quickly and easily provide the feedback.

## Why

- On-page feedback widget would make it easy for users to rate the documentation pages
- The only mechanism we currently have for users to provide feedback is to create an issue, this requirers a big investment and most readers won't commit time to do it.
- Collected feedback would provide insight into users satisfaction with docs
- Collected feedback will allow us to focus on the pages that need more work
___

## Details

**Looking for a contributor:** We are looking for a contributor that will help us fill in this proposal.

Ways to contribute: 
- If you're interested in preparing a full proposal please raise your hand 🖐️ in the discussion. 
- If you have experience and/or opinions on this topic please share them in the discussion, any input is appreciated. 

Questions to be answered:
- [ ] Define problems/solutions
- [ ] List options for integrating this in Docusaurus
- [ ] How and where would this feedback be stored?
- [ ] How can this rating/feedback be published (should be public)?
- [ ] How will this feedback be addressed?

**Comment by dgarcia360** [comment]
Hi @AleksandarCole,

I found this issue while searching for "docs feedback widget" on GitHub. I'm the founder of [PushFeedback](https://pushfeedback.com/), a feedback widget designed specifically for documentation sites that integrate well with [Docusaurus](https://docs.pushfeedback.com/installation/docusaurus).

![](https://github.com/camunda/camunda-docs/assets/9107969/ba1cdc90-6dfc-44f9-98dd-6a4126c20d21)

It's quite customizable, so you can also set it up as a widget [at the end of every page](https://docs.pushfeedback.com/installation/docusaurus#advanced-setup-custom-placement-with-swizzling):

![image](https://github.com/semaphoreci/semaphore/assets/9107969/e4f99af3-d861-4198-aa31-9433ba49dd76)

We offer a complimentary plan for open-source projects, which includes full access to our features while retaining the PushFeedback branding. If you're considering adding a feedback mechanism to your docs, I invite you to [create an account](https://app.pushfeedback.com/accounts/signup/). Once you're set up, please let me know, and I'll remove the limitations for your docs project.

Looking forward to hearing from you!

**Reply by AleksandarCole** [reply]
Thanks @dgarcia360! Looks cool and will definitely check it out. 
We are at the very early R&D with Docusaurus, but I will free free to reach out as soon as I check out PushFeedback in more detail.
